backend/src/main/java/com/redhat/cloud/notifications/routers/NotificationResource.java

```diff
@@ -183,7 +187,10 @@ public BehaviorGroup createBehaviorGroup(@Context SecurityContext sec, @NotNull
     @PUT
     @Path("/behaviorGroups/{id}")
     @Consumes(APPLICATION_JSON)
-    @Produces(APPLICATION_JSON)
+    @APIResponses(value = {
+            @APIResponse(responseCode = "200", content = @Content(mediaType = APPLICATION_JSON, schema = @Schema(implementation = BehaviorGroup.class))),
```

Comment:
actually this type is wrong and regardless that, it's not getting the media type correctly. Had to force the response.